### DISCUSSION TITLE
Question about pricing module

### DISCUSSION THREAD

**Discussion by Cydo73** [supabase/supabase#21171] [Questions]
Hello everyone. I'm new here, looking for some clarification on something. Using flutterflow and supabase. Iam building a law firm management software, but since each firm has its own cases, clients, files and staff, each firm has to get their own "app" in flutterflow or "project" in supabase. If I register under organisation on supabase on their pro plan, how many "projects" or "law firms" can I have under the plan?

**Comment by GaryAustin1** [comment]
A project is a Supabase instance.  They are completely independent and have to management separately.

A pro organization can have as many projects at a cost of additional CPU time per project.  The $25 starter includes $10 of compute time for the first project which is a micro instance.  Each additional micro instance/project is $10 for compute time.  All projects in an organization share egress/disk costs.  If you use large than micro instances your CPU costs will go up.

https://supabase.com/docs/guides/platform/org-based-billing#usage-based-billing-for-compute

**Reply by Cydo73** [reply]
Ok, thank you for the fast response, Gary. For a project with say 10-30 users, do you think the micro option would suffice

**Reply by GaryAustin1** [reply]
There is no way to answer that.  Depends on what your SQL is doing and are all users doing things at the same time.  You could have 1000 users on a micro project doing 100 simple sql operations an hour, or 20 users doing 1000 complicated queries with large amounts of data or AI vectors.  The micro instance is the same as the free instance so you can certainly do some tests.  SQL queries can be several msec or several seconds depending on what they do.
